app/analytics.go

```diff
@@ -206,7 +206,7 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 			close(commandChan)
 		}()
 
-		sessionChan := a.Srv.Store.Session().AnalyticsSessionCount()
```

Comment:
This should use the "explicit async" approach (see the example PR) using a channel and a gorouting to run it in parallel with other requests to the store.